src/EditorFeatures/CSharpTest/InlineDeclaration/CSharpInlineDeclarationTests.cs

```diff
@@ -666,8 +666,8 @@ void M()
 {
     void M()
     {
+        // prefix comment
```

Comment:
Previously we had logic that tried to move the trivia to where we were creating the out-var.  Now we just try to preserve the trivia attaching it to the next statement.